### PR TITLE
Graphs ks

### PR DESCRIPTION
**Pull request by krogowsk531**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Styling- no new features

### Detailed Description

Styling and refactoring of graphs and main display.

### Why is this change required? What problem does it solve?

Cleaner code.  Things need to be pretty!

### Files modified:
- [x] index.html
- [x] styles.css
- [x] scripts.js
- [x] activity.js
- [ ] hydration.js
- [x] sleep.js
- [ ] users.js
- [ ] testSuite.js

### CODE REVIEW

File: src/Activity.js [hunk 1]
```diff
@@ -13,18 +13,14 @@ class Activity {
...
     return userRepository.returnUserData(id);
   }
-<<<<<<< HEAD
-  weeklyActivityProperties(dateSelected, id) {
-    let startingDate = this.getDayData(dateSelected, id);
-    let firstDay = this.activitySet.indexOf(startingDate);
-    return this.activitySet.slice(firstDay, firstDay + 7).map(day => day)
-=======
+
+
```
Comment (MichaelEWalker87) [inline src/Activity.js:17]: Remove extra space

File: src/Activity.js [hunk 2]
```diff
@@ -75,24 +71,15 @@ class Activity {
...
         return
       }
-<<<<<<< HEAD
-      if (day.numSteps > perUser[i - 1].numSteps && 
+
+      if (day.numSteps > perUser[i - 1].numSteps &&
       perUser[i - 1].numSteps > perUser[i - 2].numSteps) {
         consecDays.push(day.date)
     }
-=======
```
Comment (MichaelEWalker87) [inline src/Activity.js:83]: I think this will break our test

File: src/index.html [hunk 3]
```diff
@@ -62,6 +62,7 @@ <h2 class="user-profile-display">Welcome!</h2>
     <section class="user-info-card">
     </section>
     <section class="compare-user-steps">
+      <div id="doughnutChart" style="height: 300px; width: 100%;"></div>
```
Comment (MichaelEWalker87) [inline src/index.html:65]: should we add a class?

**Comment by BlakeDonn** [comment]
We went over the changes on this pull request together on a call, and agree it moves our product to a more finished state.

File: src/scripts.js [hunk 4]
```diff
@@ -175,45 +181,47 @@ function hydrationGraph(hydrationData) {
 
 function sleepGraph(sleepData) {
   console.log(sleepData)
+  let dataPoint = sleepData.map(x => ({label: x.date, y: x.sleepQuality}))
   let sleepQualityChart = new CanvasJS.Chart("sleepChartContainer", {
+    backgroundColor: "#1D222E",
     title:{
-      text: "Your Weekly Sleep Quality Data"
+      text: "Your Weekly Sleep Quality Data",
```
Comment (MichaelEWalker87) [inline src/scripts.js:188]: looks good

**Pull request merged**